docs/src/main/sphinx/client/cli.rst

```diff
@@ -232,14 +232,17 @@ certificate usage:
     - The password for the keystore. This must match the password you specified
       when creating the keystore.
   * - ``--keystore-type``
-    - Keystore type.
+    - This advanced option is only necessary for implementing a Java
```

Comment:
Move the warning to be last sentence in this section. Also add to truststore type and reword to

```
This advanced option is only necessary if you implement a custom Java Cryptography Architecture (JCA) provider.
```